dracut.sh

```diff
@@ -1033,7 +1033,7 @@ stdloglvl=$((stdloglvl + verbosity_mod_l))
 ((stdloglvl < 0)) && stdloglvl=0
 
 [[ $drivers_dir_l ]] && drivers_dir=$drivers_dir_l
-drivers_dir="${drivers_dir%%+(/)}"
+drivers_dir="${drivers_dir%"${drivers_dir##*[!/]}"}"
```

Comment:
Alternatively, you could call `normalize_path()` on the dir. That also replaces multiple `/` inside the path by a single one, but I don't think that is a problem...

Reply:
Oh, I didn't know that exists. Probably a better solution.